models/workitemtype.go

```diff
@@ -42,11 +42,16 @@ type WorkItemType struct {
 	// Version for optimistic concurrency control
 	Version int
 	// the id's of the parents, separated with some separator
```

Comment:
This is outdated comment/description. It's not just names of the parents, it's now a full path including the name of the type itself separated by "/"